src/main/java/FileInspector.kt

```diff
@@ -0,0 +1,57 @@
+import java.io.BufferedReader
+import java.io.File
+import java.nio.file.Files
+import java.nio.file.Paths
+
```

Comment:
Praise: Very nice, clean and understandable code.
Nitpick: Lack of comments (at least an author tag would have been nice)